modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc

```diff
@@ -210,10 +210,10 @@ image::shared:byoc_apply.png[cloud_byoc_apply]
 
 Redpanda Cloud does not support the following self-hosted functionality:
 
-- OpenID Connect (OIDC) authentication
+- Data transforms (currently in beta for Redpanda Cloud)
+- Remote Read Replicas (currently in beta for Redpanda Cloud)
+- OIDC authentication
```

Comment:
@Feediver1 I think this is incorrect, OIDC is supported now: https://docs.redpanda.com/current/deploy/deployment-option/cloud/whats-new-cloud/#march-2024

cc @deniscoady

Reply:
My understanding is that we only added OIDC support for the control plane, so we still can't say we have full OIDC support. @deniscoady: Should we change this to say "OIDC authentication to the data plane"?

Reply:
If that's indeed true, I would say "Kafka API OIDC authentication. OIDC for the Cloud control plane / UI is supported".  @deniscoady to confirm

Reply:
Yes, this is true. We haven't yet done work to support OIDC authentication for Kafka API clients in the Cloud.

Reply:
clarified this item, see [here](https://deploy-preview-526--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/cloud-overview/#redpanda-cloud-vs-self-hosted-feature-compatibility)